src/sentry/models/rule.py

```diff
@@ -130,3 +131,18 @@ class RuleActivity(Model):
     class Meta:
         app_label = "sentry"
         db_table = "sentry_ruleactivity"
+
+
+def in_fourteen_days():
+    return timezone.now() + timedelta(days=14)
+
+
+@region_silo_only_model
```

Comment:
@ceorourke i think having this in the region silo makes this more difficult. You have to run the migration per-region this way. I think what we need is to run it in the control silo.